### DISCUSSION TITLE
Catching up with CodeGPT

### DISCUSSION THREAD

**Discussion by CCH44BI** [AliDehbansiahkarbon/ChatGPTWizard#21] [Ideas]
I have installed CodegPT in VS Code and am enjoying features such 

1. write documentation
2. explain selection
3. refactor selection
4. find problems
5. optimise selection

by selecting codes & right-click to bring out menu

Item 1 & 4 are actualy extremely useful.

BTW, I have extended VSCode to handle delphi by installing a Delphi extension.

**Comment by AliDehbansiahkarbon** [comment]
Well, I was thinking about more inline options like those you mentioned, but it needs a lot of requests to the API that will exceed the API rates, another story is the IDE, VsCode is an open-source IDE against the RAD Studio a close source with an undocumented API to interact with the IDE.
Anyway, I'll give it a try.
